guides/common/modules/snip_warning-destroy-vm-on-host-delete.adoc

```diff
@@ -0,0 +1,11 @@
+[WARNING]
+====
+By default, if a host entry that is associated with a virtual machine is deleted, the virtual machine will remain on the compute resource.
+
+To delete the virtual machine on the compute resource when the associated host entry is deleted:
+
+. Navigate to *Administer > Settings* and select the *Provisioning* tab.
+. Change the *Destroy associated VM on host delete* setting to *Yes*.
```

Comment:
```suggestion
If you want {Project} to delete VMs on the compute resource when the associated host entry is deleted, set the *Destroy associated VM on host delete* setting to *Yes*.
```

suggestion/question (non-blocking): Just an idea to reword this slightly. It's a snippet, so I though that instead of two steps, we could just name the relevant setting.

Because it's a snippet, I agree that it's OK to lose the Hammer CLI command compared to guides/common/modules/proc_removing-a-virtual-machine-upon-host-deletion.adoc.